src/main/java/org/fundacion/jala/converter/models/parameter/AudioParameter.java

```diff
@@ -7,12 +7,14 @@
  * license agreement you entered into with Fundacion Jala
  *
  * @author Juan Pablo Gonzales
- * @version 1.0
  */
 package org.fundacion.jala.converter.models.parameter;
 
 import org.springframework.stereotype.Service;
 
+/**
+ * This class is the options of the audio parameters.
+ */
```

Comment:
Maybe it would be better, " This class has the options for the audio... " for the javadoc of class

Reply:
ok @Paola-Xaq thanks.